app/plugin_api.go

```diff
@@ -233,6 +233,11 @@ func (api *PluginAPI) GetChannelByNameForTeamName(teamName, channelName string,
 	return api.app.GetChannelByNameForTeamName(channelName, teamName, includeDeleted)
 }
 
+func (api *PluginAPI) GetChannelStats(channelId string) (*model.ChannelStats, *model.AppError) {
+	memberCount, err := api.app.GetChannelMemberCount(channelId)
```

Comment:
I think it's idiomatic to return `nil, err` if `err != nil`, just to avoid confusion -- can you change that to make the handling explicit?